requirements.txt

```diff
@@ -3,35 +3,34 @@ attrs==23.2.0
 autobahn==23.6.2
 Automat==22.10.0
 blinker==1.7.0
-buildbot==3.11.0
-buildbot-console-view==3.11.0
-buildbot-grid-view==3.11.0
+buildbot==3.11.1
+buildbot-console-view==3.11.1
+buildbot-grid-view==3.11.1
 buildbot-profiler==1.3.1
-buildbot-waterfall-view==3.11.0
-buildbot-worker==3.11.0
-buildbot-wsgi-dashboards==3.11.0
-buildbot-www==3.11.0
+buildbot-waterfall-view==3.11.1
+buildbot-worker==3.11.1
+buildbot-wsgi-dashboards==3.11.1
+buildbot-www==3.11.1
 certifi==2024.2.2
 cffi==1.16.0
 charset-normalizer==3.3.2
 click==8.1.7
 constantly==23.10.4
-croniter==2.0.1
-cryptography==42.0.2
+croniter==2.0.2
+cryptography==42.0.5
 Flask==3.0.2
-future==0.18.3
```

Comment:
The server fails to start because of this removal.

Reply:
```
2024-03-05 15:16:05+0000 [-] Unhandled Error
	Traceback (most recent call last):
	  File "buildbot.tac", line 30, in <module>
	    m = BuildMaster(basedir, configfile, umask)
	  File "/srv/buildbot/venv/lib/python3.9/site-packages/buildbot/master.py", line 227, in startService
	    yield self._services_d
	  File "/srv/buildbot/venv/lib/python3.9/site-packages/twisted/internet/defer.py", line 2000, in _inlineCallbacks
	    result = context.run(gen.send, result)
	  File "/srv/buildbot/venv/lib/python3.9/site-packages/buildbot/master.py", line 188, in create_child_services
	    self.www = wwwservice.WWWService()
	  File "/srv/buildbot/venv/lib/python3.9/site-packages/buildbot/www/service.py", line 198, in __init__
	    self.apps = get_plugins('www', None, load_now=True)
	  File "/srv/buildbot/venv/lib/python3.9/site-packages/buildbot/plugins/db.py", line 367, in get_plugins
	    return _DB.add_namespace(namespace, interface, load_now)
	  File "/srv/buildbot/venv/lib/python3.9/site-packages/buildbot/plugins/db.py", line 317, in add_namespace
	    tempo.load()
	  File "/srv/buildbot/venv/lib/python3.9/site-packages/buildbot/plugins/db.py", line 254, in load
	    self._tree.load()
	  File "/srv/buildbot/venv/lib/python3.9/site-packages/buildbot/plugins/db.py", line 131, in load
	    child.load()
	  File "/srv/buildbot/venv/lib/python3.9/site-packages/buildbot/plugins/db.py", line 60, in load
	    self._value = self._loader(self._entry)
	  File "/srv/buildbot/venv/lib/python3.9/site-packages/buildbot/plugins/db.py", line 232, in _load_entry
	    raise PluginDBError(f'Unable to load {self._group}:{entry.name}: {str(e)}') from e
	buildbot.errors.PluginDBError: Unable to load buildbot.www:profiler: No module named 'future'
```

Fixed by re-adding `future==1.0.0`.

Reply:
Oh oh. A project doesn't announce its dependencies properly? Where does `buildbot.www:profiler` come from? The Buildbot project?

Reply:
I created https://github.com/tardyp/buildbot_profiler/issues/7 issue.

Reply:
Follow-up PR: https://github.com/python/buildmaster-config/pull/471